src/model.jl

```diff
@@ -490,48 +578,176 @@ const perihelion_parameters_phase = [251.9025, 280.8325, 128.3057, 348.1074, 292
+calc_geography_per_month_extent()
+Determines a new geography matrix by determining sea ice extent with a monthly rate of change
+and mapping extent or reduction to adjacent sea ice cells and ocean cells on the Northern Hemisphere.
+
+"""
+
+function calc_geography_per_month_extent(geography,annual_sea_ice_extent, time_step, nlongitude=128, nlatitude =65)
+
+  month::Int64 = 0
+
+ # Gets the right month when the simulation starts at the vernal equinox
+ if time_step in (1,46,47,48)
+  month = 1
+  elseif mod(time_step,4) == 2
+    month = 1
+    for nt in 1:time_step
+      if mod(nt,4) == 1
+        month +=1
+      end
+    end   
+ end
+
+  # Gets the right month when the simulation starts with the first week of January 
+  #for nt in 1:time_step
+  # if mod(nt,4) == 1
+  # month +=1
+  # end
+  #end
+  
+  nlat::Int64 = (nlatitude-1)*0.5
+  geography_start = read_geography(joinpath(@__DIR__, "..", "input", "The_World.dat"),128,65)
+  geo = geography_start[:,1:nlat]
+  sea_ice_index_unit = size(findall(isequal(2),geo),1)/annual_sea_ice_extent[1]
+  
+  
+
+  if month == 0 || month == 1
+    geography = geography_start
+    return geography
+  else
+  
+   # collect submatrix and indices for the Northern Hemisphere
+
+   G = geography[:,1:nlat]
+   N = reverse(G, dims = 2)
+   IndG = CartesianIndices(G)
+   IndN = CartesianIndices(N)
+   Gfirst, Glast = first(IndG), last(IndG)
+   Ifirst, Ilast = first(IndN), last(IndN)
+   G1 = oneunit(Gfirst)
+   I1 = oneunit(Ifirst)
+
+
+   # get all sea ice cells for the Northern Hemispehre and determine how many cells will be affected by the extent/reduction
+
+   sea_ice_index = findall(isequal(2),G)
+   old_counter = size(sea_ice_index,1)
+   new_counter = sea_ice_index_unit*annual_sea_ice_extent[month]
+   new_cells = abs(floor(Int,new_counter - old_counter))
+
+
+   # case for an extent of sea ice
+   # adjacent ocean cells will turn to sea ice cells
+
+   if new_counter > old_counter
+      for I in IndG
+          if G[I[1], I[2]] == 2
+              if new_cells <= 0
+                  break
+              else
+                  for J in max(Gfirst, I-G1):min(Glast, I+G1)
+                      if G[J] in (5,6,7,8)
+                          G[J[1], J[2]] = 2
+                          new_cells -= 1
+                         
+                      end
+                  end
+              end
+          end
+      end
+      geography[:,1:nlat] = G
+
+   # case for a reduction of sea ice
+   # adjacent sea ice cells will turn to ocean cells
+   elseif new_counter < old_counter
+      for I in IndN
+          if N[I[1], I[2]] == 2 && new_cells > 0
+              N[I[1], I[2]] = 5
+              new_cells -= 1
+              if new_cells <= 0
+                  break
+              else
+                  for J in max(Ifirst, I-I1):min(Ilast, I+I1)
+                      if N[J] == 2
+                          N[J[1], J[2]] = 5
+                          new_cells -= 1
+                      end
```

Comment:
```suggestion
                          new_cells -= 1
                          if new_cells <= 0
                              break
                          end
                      end
```
Please add this to avoid removing too many sea ice cells